scripts/metadata_comparison/metadata_comparison/lib/operation_ids.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import re
+
+papi_v1_operation_regex = re.compile('^operations/[^/]*')
```

Comment:
It's python convention to declare constants in all capital: `PAPI_V1_OPERATION_REGEX`.